1-mandatory/1-alarmclock/alarmclock.js

```diff
@@ -1,5 +1,25 @@
-function setAlarm() {}
+function setAlarm() {
+  let timeDisplay = document.getElementById("timeRemaining");
+  let alarmValue = document.getElementById("alarmSet").value;
+  let extraZero = "";
 
+  if (alarmValue > 0 && parseInt(alarmValue) === eval(alarmValue)) {
```

Comment:
😬 you need to avoid `eval`, see https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/eval

Assuming you're just trying to check if the `alarmValue` is a number, this is the appropriate way to do that:
```
const alarmNumber = parseInt(alarmValue, 10); // always specify the radix, otherwise Javascript will guess and might get it wrong
if (!Number.isNaN(alarmNumber) && alarmNumber > 0) {
    // ...
}
```